install/kubernetes/helm/istio/values.yaml

```diff
@@ -166,6 +166,7 @@ gateways:
   istio-ingressgateway:
     enabled: true
     labels:
+      app: istio-ingressgateway
```

Comment:
i wish we had used this in all gateway configs as well. If you are brave, I suggest you take a pass over all gateway configs and replace istio: ingressgateway to app: istio-ingressgateway..